tests/wpt/metadata/XMLHttpRequest/response-data-blob.htm.ini

```diff
@@ -1,5 +1,5 @@
 [response-data-blob.htm]
   type: testharness
   [XMLHttpRequest: The response attribute: Blob data]
-    expected: FAIL
+    expected: PASS
```

Comment:
This is not how we change test expectations. If it is expected to PASS, you can just delete the line. In fact, since there's only one expected test failure in this .ini file, you can delete the entire file.